src/app/educate-teacher/dashboard/components/school-details/school-details.component.scss

```
@@ -0,0 +1,40 @@
+.school-details-container {
+  .school-details-body {
+    display: flex;
+    flex-direction: column;
+
+    .detail-row {
+      display: flex;
+      align-items: baseline;
+      margin-bottom: 33px;
+
+      .label {
+        width: 232px;
+        font-weight: bold;
+        display: inline-block;
+        padding-left: 32px;
+        color: #000;
+        font-family: "Karla";
+        font-size: 24px;
+        font-style: normal;
+        font-weight: 500;
+        line-height: normal;
+      }
```

If any css property is by-default works as expected then no need to define in css file.
```suggestion
        font-style: normal;  // should remove this line
        font-weight: 500;
        line-height: normal; // as well this
      }
```